src/openforms/js/components/admin/form_design/FormLogic.stories.js

```diff
@@ -53,11 +51,13 @@ export const AVAILABLE_FORM_STEPS = [
   },
 ];
 
-<Meta
-  title="Form design/FormLogic"
-  component={FormLogic}
-  decorators={[FeatureFlagsDecorator, FormDecorator, FormLogicDecorator]}
-  parameters={{
+export default {
+  title: 'Form design/FormLogic',
+  component: FormLogic,
+  decorators: [FeatureFlagsDecorator, FormDecorator, FormLogicDecorator],
+
+  // No name supplied, should fall back to "method path (service)"
```

Comment:
If we move this comment, let's move it to line 77 `mockServiceFetchConfigurationsGet` (or an action assertion, but not in this pr... or is this covered with Chromatic?)

Reply:
right, I couldn't make much sense of this (the stories files were mostly created using the auto-migration package).

We don't have chromatic set up for the backend (yet)